### PR TITLE
added borderRadius variable to BalloonMarker in Demo

### PR DESCRIPTION
**Pull request by sweetppro**

### CODE REVIEW

**Comment by liuxuan30** [comment]
please fix the test failures.

**Comment by jjatie** [comment]
Build is failing due to:
> /Users/travis/build/danielgindi/Charts/ChartsDemo/Swift/Components/XYMarkerView.swift:21:82: missing argument for parameter 'borderRadius' in call
        super.init(color: color, font: font, textColor: textColor, insets: insets)

File: ChartsDemo/Swift/Components/BalloonMarker.swift [hunk 1]
```diff
@@ -12,71 +12,67 @@
...
-    open var insets: UIEdgeInsets
-    open var minimumSize = CGSize()
+open class BalloonMarker: MarkerImage {
+    @objc open var color: UIColor?
+    @objc open var arrowSize = CGSize(width: 15, height: 20)
+    @objc open var font: UIFont?
+    @objc open var textColor: UIColor?
+    @objc open var insets = UIEdgeInsets()
+    @objc open var minimumSize = CGSize()
+    @objc open var borderRadius = CGFloat()
```
[the remark this answers is not in the document]
Comment (sweetppro) [reply ChartsDemo/Swift/Components/BalloonMarker.swift]: this is just some values that I used in a project of mine

**Comment by petester42** [comment]
Any updates on this @sweetppro?

**Pull request closed without merging**